### DISCUSSION TITLE
How to turn off autocorrect?

### DISCUSSION THREAD

**Discussion by Asperamanca** [AnySoftKeyboard/AnySoftKeyboard#3349] [Q&A]
Sometimes, the keyboard will autocorrect words on 'space' key, despite all autocorrect settings being turned off (that is, those I could find). If that happens, an animation of the corrected word briefly floats in front of the keyboard.

- "Quick fixes and abbreviations" is turned off
- Spell checker in Android settings is off
- Autofill service in Android settings is off
...and that was all I could find

How do I fully turn off autocorrect?

**Comment by revnu** [comment]
Same here. There is no option to disable autocorrect then space is pressed. The only way is to disable the whole feature, but I want to see suggestions on top bar.

**Comment by alchemiker** [comment]
Indeed there is an option for this, I made this on my phone, too. You have to go to _Language_ > _Configure correction behaviour and other dictionaries_ and in the Section _Grammar_ to tap on _Auto-select suggestion aggressiveness_. Then you set the Aggressiveness level to "Off".
Then the app will keep suggesting you the words, but won't select them automatically and insert them only if you deliberately tap on them.
